gov.pnnl.goss.gridappsd/src/gov/pnnl/goss/gridappsd/simulation/SimulationProcess.java

```diff
@@ -153,8 +158,19 @@ public void run() {
 						GridAppsDConstants.topic_platformLog);
 
 
-				//Send the timesteps by second for the amount of time specified in the simulation config
-                sendTimesteps(gossFncsResponseEvent, simulationConfig, simulationId);
+				//Send the start simulation command to the fncsgossbridge
```

Comment:
Seems as if this comment is really far to the right...?

Reply:
Well that's freaking weird. they are lined up in my eclipse editor. must be a spaces vs tabs issue?